surveys/promise-rejections/survey.md

```diff
@@ -88,6 +88,30 @@ As we can see in the examples, an unhandled rejection might be handled in the fu
 
 Certain unhandled rejections may in rare cases leave your application in a non-deterministic and unsafe state, whether it's internal application state (including memory leaks), external resources used by your application (say, file handles or database connections), or external state (say, consistency of data in a database).
 
+As an example, the following server is not sending a response back to the client, causing a socket leak and a possible Denial of Service attack:
+
+```js
+const http = require('http')
+const server = http.createServer(handle)
+
+server.listen(3000)
+
+function handle (req, res) {
+  doStuff()
+    .then((body) => {
+      res.end(body)
+    })
+}
+
+function doStuff () {
+  if (Math.random() < 0.5) {
+    return Promise.reject(new Error('kaboom'))
+  }
+
+  return Promise.resolve('hello world')
+}
```

Comment:
total nit, but this way it'll match the most common style in the ecosystem
```suggestion
const http = require('http');
const server = http.createServer(handle);

server.listen(3000);

function handle(req, res) {
  doStuff()
    .then((body) => {
      res.end(body);
    });
}

function doStuff() {
  if (Math.random() < 0.5) {
    return Promise.reject(new Error('kaboom'));
  }

  return Promise.resolve('hello world');
}
```

Reply:
~oh no, we'll start discussing semicolons now~

Reply:
I should apply a linter here or something, I just noticed there's no consistency across the examples :/

Reply:
Ran eslint on code blocks, should be more following a single style now